digital-deck-ui/src/components/PlayerHand.vue

```diff
@@ -1,12 +1,37 @@
 <template>
-    <div>
-        <div>Display player's cards here</div>
-        <div>Current hand (For testing): {{ $store.getters.getPlayerCards }}</div>
+  <div id="player-area">
```

Comment:
Are you using this id anywhere?

Reply:
I put this so it was easier to tell player-area when dubbing with inspect. No need for it now